source/monitoring.txt

```diff
@@ -35,6 +35,104 @@ To learn more about logging in the connector, see the
 
 .. _kafka-monitoring-information:
 
+Use Cases
+---------
+
+This section describes use cases for monitoring MongoDB source and sink
+connectors.
+
+Sink Connector
+~~~~~~~~~~~~~~
+
+The following table describes some use cases for monitoring the MongoDB
+sink connector and the metrics the sink connector provides
+to satisfy those use cases:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use Case
+     - Metrics to Use
+
+   * - You need to know the total number of records your connector
+       wrote to MongoDB.
+     - Use the ``records`` metric.
+
+   * - You need to know how long the interval of time is between
+       when a record arrives in a Kafka topic and when your connector writes
+       that record to MongoDB.
+     - Use the ``latest-kafka-time-difference-ms`` metric.
+
+   * - You need to know the total number of write errors your connector
+       encountered when attempting to write to MongoDB.
+     - Use the ``batch-writes-failed`` metric.
+
+   * - You need to know if your MongoDB performance is getting slower
+       over time.
+     - Use the ``getmore-commands-successful-duration-over-<number>-ms``
+       and ``getmore-commands-failed-duration-over-<number>-ms``
+       metrics. Track the change in these metrics over
+       time. See if the number of longer running commands is increasing
+       at a faster rate than the number of shorter running commands.
+ 
+   * - You need to find a bottleneck in how {+kc+} and your MongoDB sink
+       connector write {+ak+} records to MongoDB.
+     - Compare the values of the following metrics:
+     
+       - ``in-task-put-duration-ms``
+       - ``in-connect-framework-duration-ms``
+       - ``processing-phase-duration-ms``
+
+You can find descriptions of all MongoDB sink connector metrics in the
+:ref:`<kafka-monitoring-sink-attributes>` section.
+
+Source Connector
+~~~~~~~~~~~~~~~~
+
+The following table describes some use cases for monitoring the MongoDB
+source connector and the metrics the source connector provides
+to satisfy those use cases:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use Case
+     - Metrics to Use
+
+   * - You need the number of change stream events your source connector
+       has processed. 
+     - Use the ``records`` metric.
+
+   * - You need to know how old the most recent change stream event your connector
+       processed is.
+     - Use the ``latest-mongodb-time-difference-secs``
+       metric.
+
+   * - You need to know the percentage of records your connector
+       received but failed to write to {+ak+}.
+     - Use the ``records-filtered`` and ``records`` metrics.
+
+   * - You need to know the average size of the documents your connector
+       has processed.
+     - Use the ``mongodb-bytes-read`` and ``records`` metrics.
```

Comment:
Maybe explicitly say to divide bytes by records (rather than "use"). 

This is also a place where users might care about a span of time, which I worried might be confusing (as you explain elsewhere, one needs to perform T2-T1 to get the number over a duration, and then perform the math.)

Reply:
Explicitly stated formula reader should use and linked to additional admonition later in page